unfetter-discover/unfetter-ui
# Issue 1146/blsummary weightings

Fixes unfetter-discover/unfetter#1146

Weighting completion percentages for protect, detect, and respond have been added in for the baseline summary page.

### To test:
1. Pull this branch
2. Create a new baseline or edit an existing baseline
3. Note percentages for P, D, and R - should be 100% in all cases*

*Percentages for protect, detect, and respond will always report 100% at this point until the fix for separating None and Unanswered is complete (see Stix PR https://github.com/unfetter-discover/stix/pull/25 and unfetter-store PR https://github.com/unfetter-discover/unfetter-store/pull/214)
